wavenet/audio_reader.py

```diff
@@ -105,14 +169,22 @@ def thread_main(self, sess):
                 if self.sample_size:
                     # Cut samples into fixed size pieces
                     buffer_ = np.append(buffer_, audio)
-                    while len(buffer_) > self.sample_size:
+                    while len(buffer_) > 0:
```

Comment:
Hi @jyegerlehner , by changing to this, did you mean that we will pad 0 at the end of the `buffer` if length of `buffer` < length of `sample_size`?